setup.py

```diff
@@ -49,9 +49,11 @@
     ],
     'postgresql': [
         'invenio-db[postgresql,versioning]>=1.0.0a9',
+        'psycopg2',
```

Comment:
it does not seem to be so: https://travis-ci.org/inveniosoftware/invenio-workflows/jobs/147408433